src/frontend/src/lib/canisters/backend.errors.ts

```diff
@@ -0,0 +1,27 @@
+import type {
+	BtcAddPendingTransactionError,
+	SelectedUtxosFeeError
+} from '$declarations/backend/backend.did';
+import { CanisterInternalError } from '$lib/canisters/errors';
+
+export const btcPendingTransactionError = (response: BtcAddPendingTransactionError) => {
+	if ('InternalError' in response) {
+		return new CanisterInternalError(response.InternalError.msg);
+	}
+
+	return new Error('Unknown BtcAddPendingTransactionError');
+};
+
+export const btcSelectUserUtxosFeeError = (response: SelectedUtxosFeeError) => {
```

Comment:
In `ic-js` we often prefix those function to map error with the keyword, `map`. Should we do the same here?

e.g. `mapBtcSelectUserUtxosFeeError`

Reply:
Sounds good, done 👍